components/ui/widgets/src/main/res/values/attrs.xml

```diff
@@ -6,4 +6,7 @@
     <attr name="mozac_font_semibold" format="reference" />
     <attr name="mozac_accent" format="reference" />
     <attr name="mozac_contrast_text" format="reference" />
+
+    <attr name="mozac_widget_favicon_background" format="reference" />
+    <attr name="mozac_widget_favicon_border" format="reference" />
```

Comment:
How do you feel about calling this `mozac_widget_favicon_border_color`?

Reply:
Sounds good, I'm happy to make the attribute names more descriptive.